sdk/storage/azure-storage-file-datalake/swagger/src/main/java/DataLakeStorageCustomization.java

```diff
@@ -14,23 +14,28 @@ public class DataLakeStorageCustomization extends Customization {
     @Override
     public void customize(LibraryCustomization customization, Logger logger) {
         PackageCustomization implementationModels = customization.getPackage("com.azure.storage.file.datalake.implementation.models");
-
         implementationModels.getClass("BlobHierarchyListSegment")
             .addAnnotation("@JsonDeserialize(using = com.azure.storage.file.datalake.implementation.util.CustomHierarchicalListingDeserializer.class)");
+        changeJsonPropertyValue(implementationModels.getClass("FileSystemList"), "filesystems", "filesystems");
```

Comment:
What caused this change?

Reply:
This was done to harden code generation now that Autorest is validated as part of PRs. It was found there is race conditions in customizations where very rarely these changes would result in an import being dropped, causing a compile issue later on.